playgrounds/vue/README.md

```diff
@@ -0,0 +1,40 @@
+# vue
```

Comment:
Minor suggestion.

It looks like the auto-generated readme.
It doesn't useful in our current mono repo structure.
Maybe we should change or delete it? :)

Reply:
done